package.json

```diff
@@ -74,7 +74,7 @@
     "@langchain/community": "^0.3.4",
     "@langchain/core": "^0.2.9",
     "@langchain/mistralai": "^0.0.28",
-    "@upstash/vector": "^1.1.3",
```

Comment:
We should change the peer dependency version instead, this does not prevent the user from supplying their own version of upstash vector, which does not have hybrid.

It's better for us to get a proper release first, then add vector as just a peer dependency (they get installed by default anyways so nothing should change).